scout/constants/file_types.py

```diff
@@ -2,11 +2,13 @@
 
 FILE_TYPE_MAP = {
```

Comment:
Make this one an OrderedDict?

Reply:
That was my first thought, but it won't really help since we then add fields from the member dicts to separate sets. Its those that need to be ordered.

Reply:
While it works it feels that the solution relies entirely on the fact that the naming of the variables, that are OK by chances. What is we need to introduce new categories in the future or we forget why we introduced that sorting? 🤔

Reply:
Well, it's more that it is the same as before than that it is ok: there is really no way to say a 52 bp variant should rather be an sv or an snv.

Reply:
Agreed, but I guess it's also OK to define an arbitrary order, because that's what all is about

Reply:
Ok, here goes. With this, we are now free to deviate from alphabetical order, and until we have one that does this could also potentially compete in a contest for obfuscated python. 😸

Reply:
Thanks, I'll review first thing tomorrow!